src/content/docs/docs/intro/web-monetization-extension.mdx

```diff
@@ -0,0 +1,225 @@
+    src="/img/docs/extension/otp-success.png"
+    alt="Extension showing an available balance of four dollars, with a success message indicating one dollar was sent"
+    style="max-width:300px"
+    class="img-outline"
+    />
+</Steps>
+
+## Disconnect the extension from your wallet
+
+When you disconnect the extension from your wallet, the extension can no longer make payments. You can always reconnect your wallet later.
+
+:::tip[Remaining balance]
+You won't lose your remaining balance if you disconnect the extension from your wallet. Your remaining balance will remain in your wallet account. 
+:::
+
+The extension only supports one wallet connection at a time. If, for example, you have two wallet accounts from which you want to make payments, you must disconnect from the first wallet before you can connect the extension to your second wallet. 
+
+<Steps>
+1. Click the **Settings** icon in the upper-right of the extension.
+    <br /><img src="/img/docs/extension/settings-icon.png" alt="Extension showing a cog icon in the upper-right" style="max-width:300px" class="img-outline"/>
+2. Click **Disconnect**.
+    <br /><img
+    src="/img/docs/extension/wallet-tab.png"
+    alt="Extension's wallet tab showing the disconnect button"
+    style="max-width:300px"
+    class="img-outline"
+    />
+    The extension's setup screen appears with your previously used wallet address and budget settings prefilled.
+    <img
+    src="/img/docs/extension/connection-setup-filled.png"
+    alt="Extension's setup screen with previously used wallet address, budget amount, and recurrence setting saved"
+    style="max-width:300px"
+    class="img-outline"
+    />
+</Steps>
+
+## Resolve a key addition failure <a id="manually-add-key"></a>
+
+When you try to connect the extension to your digital wallet, the extension attempts to automatically connect with your wallet provider to add a key to your wallet account. 
+
+You'll receive an error if the extension is unable to add the key. 
+
+<img
+    src="/img/docs/extension/connection-failed.png"
+    alt="Extension showing an automatic connection error, the key you must copy, and a copy button for the key"
+    style="max-width:300px"
+    class="img-outline"
+    />
+
+When this happens, you must copy the key that appears in the extension and manually add it to your wallet account. The steps for adding the key to your account will vary between wallet providers.
+
+<StylishHeader>Interledger Wallet</StylishHeader>
+
+Follow these steps if your wallet provider is Interledger wallet. 
+
+<Steps>
+1. Sign in to your Interledger Wallet.
+2. Select **Settings** from the left menu.
+3. Select **Keys**.
+4. Click **Add a public key**.
+5. Enter a nickname for the key, then paste the key into the public key field.
+6. Click **Save**.
+7. Go back to your extension and try connecting again.
+</Steps>
+
+Contact Interledger Wallet at support@interledger.app if you need additional help.
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Nit: Why is this whitespace here? 😄

Reply:
Ah, I was using the space kind of like a notepad. Thanks for calling it out.